core/embed/rust/src/ui/model_tt/layout.rs

```diff
@@ -2,20 +2,34 @@ use core::convert::{TryFrom, TryInto};
 
 use crate::{
     error::Error,
-    micropython::obj::Obj,
+    micropython::{buffer::Buffer, map::Map, obj::Obj, qstr::Qstr},
     ui::{
-        component::{Child, FormattedText},
+        component::{
+            base::ComponentExt, text::paragraphs::Paragraphs, Child, FormattedText, PageMsg,
+        },
         display,
         layout::obj::LayoutObj,
     },
     util,
 };
 
 use super::{
-    component::{ButtonMsg, DialogMsg, HoldToConfirm, HoldToConfirmMsg},
+    component::{Button, ButtonMsg, DialogMsg, HoldToConfirm, HoldToConfirmMsg, SwipePage, Title},
     theme,
 };
 
+impl<T> TryFrom<PageMsg<T, bool>> for Obj {
```

Comment:
vscode is notifying me of "conflicting implementation" when t1 and tt are both enabled -- which indicates that this might be better moved to common code?

Reply:
Wanted to solve this as part of "sane rs->py message passing" but this one is actually semi-sane, so fixed in 0292fbfee125d7ed2f201c52447e074c9a1bb2ad.